corehq/ex-submodules/phonelog/migrations/0005_add_forceclose_entry_20160408_1530.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('phonelog', '0004_merge'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='ForceCloseEntry',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('domain', models.CharField(max_length=100, db_index=True)),
```

Comment:
Okay, so given that the index is already there, it  seems like dropping it makes the most sense (the table will be empty or near-empty at this point anyways).  

I see mine also has the same string in between `domain_` and `_like`, is that a guarantee?  If so, I'll just add in another migration like the second one you proposed.